test/twine_test.coffee

```diff
@@ -644,43 +642,33 @@ suite "Twine", ->
   suite "data-allow-default", ->
     test "should prevent default action for an anchor tag", ->
       testView = "<a data-bind-event-click=\"fn()\"></a>"
-      event = {type: 'click', preventDefault: @spy()}
       node = setupView(testView, fn: ->)
 
-      jQuery(node).trigger(event)
-      assert.isTrue event.preventDefault.calledOnce
+      assert.isFalse triggerEvent(node, "click")
```

Comment:
This took a while to figure out how to resolve but after some searching the [MDN `dispatchEvent` docs](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent) revealed that the result of `dispatchEvent` will be:

> The return value is false if event is cancelable and at least one of the event handlers which received event called Event.preventDefault(). Otherwise it returns true.

This works perfectly for our scenario as the previous tests were asserting the dummy "event" object's spy was called.